client/settings/use-payment-method-availability/index.tsx

```diff
@@ -0,0 +1,205 @@
+	const paymentMethodStatuses = useGetPaymentMethodStatuses();
+	const [ enabledPaymentMethods ] = useEnabledPaymentMethodIds();
+	const [ isManualCaptureEnabled ] = useManualCapture();
+
+	const isPoEnabled = wcpaySettings?.progressiveOnboarding?.isEnabled;
+	const isPoComplete = wcpaySettings?.progressiveOnboarding?.isComplete;
+
+	const {
+		stripe_key: stripeKey,
+		currencies,
+		label,
+		allows_manual_capture: isAllowingManualCapture,
+	} = methodsConfiguration[ id ];
+
+	const { status } = paymentMethodStatuses[ stripeKey ] ?? {
+		status: upeCapabilityStatuses.UNREQUESTED,
+		requirements: [],
+	};
+
+	// We want to show a tooltip if PO is enabled and not yet complete. (We make an exception to not show this for card payments).
+	const isPoInProgress = isPoEnabled && ! isPoComplete;
+	if (
+		upeCapabilityStatuses.INACTIVE === status ||
+		( isPoInProgress && upeCapabilityStatuses.UNREQUESTED === status )
+	) {
+		return {
+			isActionable: false,
+			chip: __( 'More information needed', 'woocommerce-payments' ),
+			notice: interpolateComponents( {
+				// translators: {{learnMoreLink}}: placeholders are opening and closing anchor tags.
+				mixedString: __(
+					'We need more information from you to enable this method. ' +
+						'{{learnMoreLink}}Learn more{{/learnMoreLink}}',
+					'woocommerce-payments'
+				),
+				components: {
+					learnMoreLink: (
+						// eslint-disable-next-line jsx-a11y/anchor-has-content
+						<ExternalLink
+							title={ __(
+								'Learn more about enabling payment methods',
+								'woocommerce-payments'
+							) }
+							href={ getDocumentationUrlForDisabledPaymentMethod(
+								id,
+								isPoInProgress
+							) }
+						/>
+					),
+				},
+			} ),
+		};
+	}
+
+	if ( upeCapabilityStatuses.PENDING_APPROVAL === status ) {
+		const paymentMethodsWithDelayedApproval: string[] = [
+			PAYMENT_METHOD_IDS.ALIPAY,
+			PAYMENT_METHOD_IDS.WECHAT_PAY,
+		];
+
+		return {
+			isActionable: false,
+			chip: __( 'Approval pending', 'woocommerce-payments' ),
+			notice: paymentMethodsWithDelayedApproval.includes( id )
+				? sprintf(
+						__(
+							'%s requires your store to be live and fully functional before it can be reviewed for use with their service. This approval process usually takes 2-3 days.',
+							'woocommerce-payments'
+						),
+						label
+				  )
+				: __(
+						'This payment method is pending approval. Once approved, you will be able to use it.',
+						'woocommerce-payments'
+				  ),
+		};
+	}
+
+	if ( upeCapabilityStatuses.PENDING_VERIFICATION === status ) {
+		return {
+			isActionable: false,
+			chip: __( 'Pending verification', 'woocommerce-payments' ),
+			notice: sprintf(
+				__(
+					"%s won't be visible to your customers until you provide the required " +
+						'information. Follow the instructions sent by our partner Stripe to %s.',
+					'woocommerce-payments'
+				),
+				label,
+				wcpaySettings?.accountEmail ?? ''
```

Comment:
```suggestion
				wcpaySettings?.accountEmail ?? 'your email'
```

Reply:
I _think_ the issue of the missing email address mainly happens on local environment, I don't remember seeing it on JN (for example).
In any case, I think that's fair, I made the change in e08cb246d with a slight tweak for translators.